apps/notifications/src/panel/templates/actions.jsx

```diff
@@ -13,6 +14,8 @@ import ReplyInput from './comment-reply-input';
 
 const getType = ( note ) => ( null === getReferenceId( note, 'comment' ) ? 'post' : 'comment' );
 
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
```

Comment:
would be better to do `esling-disable-next-line` here, or move this to the top of the file.

Reply:
Moved to top of file